packages/atti-components/src/components/Stepper/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import Step from './styles/StyledStep'
+import StyledLabel from './styles/StyledLabel'
+import { StepperProvider } from './hooks'
+
+/**
+ * A basic stepper
+ */
+
+const Stepper = ({ variant, children }) => {
+  return <StepperProvider variant={variant}>{children}</StepperProvider>
+}
+
+Stepper.defaultProps = {
+  variant: 'number',
+}
+
+Stepper.propTypes = {
+  /**
+   * The variant of the stepper.
+   */
+  variant: PropTypes.oneOf(['number', 'checks', 'normal']),
+  children: PropTypes.any.isRequired,
+}
+
+Stepper.Step = Step
+Stepper.Label = StyledLabel
+
+/** @component */
+export default styled(Stepper)``
```

Comment:
Responsive can be improved too. There are multiple possibilities. Here are some examples:
https://codepen.io/5t3ph/pen/wvwbNqx
https://material.angular.io/components/stepper/overview
On the last one, the stepper becomes vertical. IMO it's the easiest / best solution even if we have a huge number of items.

Reply:
The stepper is now responsive, thanks for your help :)